broker/pkg/model/config/store.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:generate sh -c ./generate_mock.sh
```

Comment:
should this be a go:generate on the test file?
It is not needed by store.go

Reply:
gomock completely removed from these files altogether now. unnecessary dependency that had leaked across packages.